### DISCUSSION TITLE
Ability to configure tree default queries and counts

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5701] [Feature Requests]
> Is there a way for the Tree to display counts for all determinations (not just current), or configure the auto-query to automatically search ALL dets?

In this instance, the workflow requires seeing if **any** dets are associated with a tree node. There is no way to configure this at the moment.

**Requested By:** [@HeatherC on the Community Forum](https://discourse.specifysoftware.org/t/records-missing-from-taxon-tree-counts/947/3?u=specify)

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/records-missing-from-taxon-tree-counts/947/4

**Comment by grantfitzsimmons** [comment]
> Since this is implicitly related, I thought I would add here, instead of starting a new thread.
> 
> I am spending some time ‘pruning’ our taxon tree. Often that means I am using the query function to review records connected to a node. When I do this, I want to see ‘all’ of them, not just those with current dets. Is there a way that I can configure my default preferences for this taxon query to be ‘any’ instead of ‘True or Empty’?
>
> ![image](https://user-images.githubusercontent.com/37256050/212100076-aeebf5b8-cbf0-4bf4-9420-4aa6f4a74170.png)
> 
> It isn’t too bad to edit the query each time, BUT once I have edited the query, I ALSO get the ‘Leave warning’ (which I often appreciate, but not when I have to click ‘leave’ for every query I am doing, cause I have to edit every query). Having to do the extra clicks to get the info I want AND to close the page starts adding up.
> 
> ![image](https://user-images.githubusercontent.com/37256050/212100136-efca260f-b847-48f2-a0af-030fd1162cc1.png)
>
> https://discourse.specifysoftware.org/t/records-missing-from-taxon-tree-counts/947/5?u=specify

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/records-missing-from-taxon-tree-counts/947/6

**Comment by maxpatiiuk** [comment]
Is this a duplicate of #1224?
Closing that one as this has more information

**Comment by grantfitzsimmons** [comment]
@maxpatiiuk Yes it is

**Comment by grantfitzsimmons** [comment]
> Tree view has a "Query" button that opens a hardcoded query for a given tree.
> @acbentley suggested promoting the user to select an existing query.
> Alternatively, could allow people to configure the default query instead of showing a hardcoded one

Copied from #1224
